corehq/motech/dhis2/const.py

```diff
@@ -66,3 +65,17 @@
 XMLNS_DHIS2 = 'http://commcarehq.org/dhis2-integration'
 
 DHIS2_MAX_VERSION = "2.35.1"
+
+COMPLETE_DATE_EMPTY = "complete_date_empty"
+COMPLETE_DATE_COLUMN = "complete_date_column"
+COMPLETE_DATE_ON_PERIOD_END = "complete_date_on_period_end"
+COMPLETE_DATE_ON_SEND = "complete_date_on_send"
+
+COMPLETE_DATE_CHOICES = [
+    (COMPLETE_DATE_EMPTY, _('None')),
+    (COMPLETE_DATE_COLUMN, _('UCR column')),
+    (COMPLETE_DATE_ON_PERIOD_END, _('Use last day of period')),
+    (COMPLETE_DATE_ON_SEND, _('Use date that dataValues are sent')),
+]
+DAYS_IN_ONE_WEEK = 7
+MONTHS_IN_ONE_QUARTER = 3
```

Comment:
Nit: I don't think you need to define constants for these, because the meaning of the values will be obvious in the code, especially considering that you've used good variable names in the places where you are using these constants.

Reply:
Yeah I also thought this may be a bit over the top, but did it anyway. Since there's another (aka yourself) hinging toward not putting it here, I'll drop it.